src/main/resources/org/jenkinsci/plugins/nomad/NomadSlaveTemplate/config.jelly

```diff
@@ -16,6 +16,10 @@
             <f:textbox/>
         </f:entry>
 
+        <f:entry title="${Constraints}" field="constraints" description="comma separated LTarget,RTarget,Operand">
```

Comment:
Well the commas interfere with `set_contains`, maybe it would be better to do something like:
ltarget operand rtarget|ltarget operand rtarget
I also need to find some way to be able to deal with ${ } as well, I will take a look again to see if its possible to accept the input for ${ } directly in the textbox, or maybe prepend ltarget with a character such as `-` to allow the expansion to happen, and to keep things somewhat future-proof